pokecli.py

```diff
@@ -453,11 +444,8 @@ def task_configuration_error(flag_name):
             task_configuration_error('{}.{}'.format(outer, inner))
             return None
 
-    if (config.evolve_captured
-        and (not isinstance(config.evolve_captured, str)
-             or str(config.evolve_captured).lower() in ["true", "false"])):
-        parser.error('"evolve_captured" should be list of pokemons: use "all" or "none" to match all ' +
-                     'or none of the pokemons, or use a comma separated list such as "Pidgey,Weedle,Caterpie"')
+    if config.evolve_captured:
+        parser.error('"evolve_captured" has been removed in favor of the EvolveTask')
```

Comment:
Can we make this a deprecation warning that lets the bot keep running? I'd like to avoid breaking peoples' configs if possible.